docs/content/preview/explore/ysql-language-features/pg-extensions/extension-pgpartman.md

```diff
@@ -327,3 +327,70 @@ mo=# SELECT create_parent( p_parent_table => 'public.orders',
 ```output
 ERROR:  Partition table public.orders is a colocated table hence registering it to pg_partman maintenance is not supported
 ```
+
+### xCluster
+
+Depending on the type of xCluster deployment, pgPartman may be used.
+
+Note that if you are [using pg_cron to manage tasks](#maintain-partitions-using-pg-cron), you must install it on a separate database that is not part of the xCluster configuration.
+
+For more information on xCluster limitations, refer to [Limitations](../../../../architecture/docdb-replication/async-replication/#limitations).
+
+#### Transactional xCluster and xCluster DR
+
+**Automatic mode**
+
+The pgPartman maintenance cron job can only be enabled on the xCluster primary. Disable it on the xCluster standby.
+
+During disaster recovery switchover (or failover and repair):
+
+1. Disable the pgPartman cron job on the original xCluster primary before initiating switchover.
+1. After switchover or failover, enable the pgPartman cron job on the new primary.
+
```

Comment:
From 2025.1.1 - 
This can be replaced with: pg_cron will only execute on the Primary universe. On switchover pg_cron will automatically switchover to the new Primary universe as well.